sakai-evaluation-tool/src/java/org/sakaiproject/evaluation/tool/reporting/ReportExporterBean.java

```diff
@@ -45,6 +45,14 @@
 
     private static Log log = LogFactory.getLog(ReportExporterBean.class);
 
+    // Section awareness/new report style bindings
+    public String viewID            = null;
```

Comment:
Just tested it out, seems like it's happy with empty strings/arrays. Consider it done.